telegram

```diff
@@ -268,6 +269,11 @@ while getopts "t:c:i:f:V:MHCrhlvjnRmDNT:" opt; do
 		T)
 			TITLE="$OPTARG"
 			;;
+                I)
+                        TELEGRAM_IP=`dig @8.8.8.8 api.telegram.org | grep "ANSWER SECTION" -A 2 | grep -v -e CNAME  -e "ANSWER SECTION" | cut -f5`
```

Comment:
What about using `dig api.telegram.org @8.8.8.8 +short`? Then there's no need to use grep and cut.

Reply:
I was not aware of this option. Please see this PR in which I fixed the issue: https://github.com/fabianonline/telegram.sh/pull/35

I've also closed this one since it was not on purpose for me to update my fork specific readme within your repository.